doc/ib-vf-guid-configuration.md

```diff
@@ -0,0 +1,80 @@
+# Infiniband VF GUID Configuration
```

Comment:
nit: Infiniband VF GUID Static Configuration

also please rename file to : ib-vf-guid-static-configuration.md